smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/CreatePageAsAuthorUserIT.java

```diff
@@ -68,16 +111,16 @@
     @Test
     public void testCreatePageAsAuthor() throws InterruptedException, ClientException {
         String pageName = "testpage_" +  UUID.randomUUID();
-        String pagePath = temporaryPage.getParentPath() + "/" + pageName;
+        String pagePath = "/content/" + pageName;
```

Comment:
This is really not the way to go, creating pages directly under `/content` or hardcode it. The `Page` rule has a homogenous way of creating content under a test site it always provisions if missing. These tests should be able to run on any instance of AEM, with no assumptions about existing users and they need to be safe to execute against a production environment.

Reply:
I was thinking the same, but the test site is not getting initialised without the `cqRuleChain` rule which is dependant on `TemporaryUser` rule.